robotoff/metrics.py

```diff
@@ -74,6 +74,23 @@ def get_influx_client() -> InfluxDBClient:
     )
 
 
+def ensure_influx_database():
+    client = get_influx_client()
+    if client is not None:
+        try:
+            db_names = [data.get("name") for data in client.get_list_database()]
+            if settings.INFLUXDB_DB_NAME not in db_names:
+                # create it
+                client.create_database(settings.INFLUXDB_DB_NAME)
+                logger.warning(
+                    "Creating influxdb database %r as it does not exists yet",
```

Comment:
it does not exists yet -> it does not exist yet

Reply:
fixed